plugin.py

```diff
@@ -58,11 +66,50 @@ def platform() -> str:
         return "unknown-linux-gnu"
 
 
+def open_runnables_in_terminus(window: Optional[sublime.Window], runnables: List[Runnable]) -> None:
+    filtered_runnables = [r for r in runnables if r["kind"] == "cargo"]
+    if len(filtered_runnables) == 0:
+        return
+    if not window:
+        return
+    view = window.active_view()
+    if not view:
+        return
+    if not Terminus:
+        sublime.error_message(
+            'Cannot run executable. You need to install the "Terminus" package and then restart Sublime Text')
+        return
+    main_cargo = shutil.which("cargo")
+    if not main_cargo:
+        sublime.error_message('Cannot find "cargo" on path.')
+        return
+    main_cargo_path = '"{}"'.format(main_cargo)
+    for runnable in filtered_runnables:
+        if runnable["args"]["overrideCargo"]:
+            cargo_path = runnable["args"]["overrideCargo"]
+        else:
+            cargo_path = main_cargo_path
+        command_to_run = [cargo_path] + runnable["args"]["cargoArgs"]
+        if runnable["args"]["cargoExtraArgs"]:
+            command_to_run += runnable["args"]["cargoExtraArgs"]
+        if runnable["args"]["executableArgs"]:
+            command_to_run += ['--'] + runnable["args"]["executableArgs"]
+        args = {
+            "title": runnable["label"],
+            "shell_cmd": " ".join(command_to_run),
```

Comment:
Generally speaking, it's more correct to use `shlex.join(command_to_join)` since it'd handle the quoting aspect for you. I'm unsure however whether the arguments in the runnable are quoted already.

Reply:
I've raised this myself in the original PR but yes `shlex.join` is python 3.8+.

The LSP and the helper packages run in 3.3 right now. Being able to switch to 3.8 is still fairly far away due to dependencies not being supported there yet.

Reply:
Using `shell_cmd` for Terminus is silly anyway, why not use Terminus' `cmd` argument and pass the list as-is? Then there's no need for this shlex stuff.

Reply:
Changed to `cmd` and it seemed to work when I've tested.